Project-Service-implementation-Script/main.py

```diff
@@ -135,6 +141,10 @@ def programCreation(accessToken,parentFolder,externalId,pName,pDescription,roles
       "language" : [ 
           "English"
       ],
+      "metaInformation" : {
+        "state" : [entities],
+        "roles" : [role]
```

Comment:
@MallanagoudaB what are the sample values for role?

Reply:
state roles